ios/magnet/AppDelegate.m

```diff
@@ -52,8 +54,17 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
   UIViewController *rootViewController = [UIViewController new];
   rootViewController.view = rootView;
   self.window.rootViewController = rootViewController;
+  [NotificationsHelper register];
   [self.window makeKeyAndVisible];
   return YES;
 }
 
+- (void)applicationDidBecomeActive:(UIApplication *)application {
+  [NotificationsHelper clearNotifications];
+}
+
+- (void)applicationDidEnterBackground:(UIApplication *)application {
+  [NotificationsHelper clearNotifications];
```

Comment:
Is there a way to know if the app is in the foreground so we can prevent the notifications being dispatched?